New version: wenming.ThirdSpace version 0.2.0

wenming-ma opened the pull request:
### New Version

- Package: wenming.ThirdSpace
- Version: 0.2.0

### What's New
- Added autostart on boot option in Advanced Settings

### URLs
- GitHub: https://github.com/wenming-ma/ThirdSpace
- Release: https://github.com/wenming-ma/ThirdSpace/releases/tag/v0.2.0
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/325399)

wenming-ma committed:
New version: wenming.ThirdSpace version 0.2.0

Trenly commented on manifests/w/wenming/ThirdSpace/0.2.0/wenming.ThirdSpace.locale.en-US.yaml:
```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: wenming.ThirdSpace
+PackageVersion: 0.2.0
+PackageLocale: en-US
+Publisher: WenMing
```
```suggestion
Publisher: thirdspace
```
This should match what is written to control panel, or AppsAndFeaturesEntries should be specified in the installer manifest

wenming-ma committed:
Update manifests/w/wenming/ThirdSpace/0.2.0/wenming.ThirdSpace.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

wenming-ma committed:
Update Publisher to match Control Panel entry

wenming-ma committed:
Merge branch 'wenming-ThirdSpace-0.2.0' of https://github.com/wenming-ma/winget-pkgs into wenming-ThirdSpace-0.2.0

wenming-ma committed:
Add AppsAndFeaturesEntries and revert Publisher to WenMing

wenming-ma committed:
Update Publisher to match MSI Manufacturer

- Change Publisher to 'thirdspace' in locale manifest
- Update MSI URL and SHA256 for rebuilt installer
